modules/installation-guide/partials/proc_configuring-workspace-exposure-strategies-using-a-helm-chart.adoc

```diff
@@ -34,9 +34,9 @@ The supported values for `global.serverStrategy` are:
 * xref:default-host-workspace-exposure-strategy_{context}[`default-host`]
 
 
-.Gateway single-host on Kubernetes
+.Gateway single-host on {orch-name}
```

Comment:
Shouldn't we keep `Kubernetes` here since we're in the context of installing Che from helm charts, which is upstream-only and not targeted at OpenShift installations ?